package.json

```diff
@@ -75,39 +75,41 @@
     "**/*": "prettier --write --ignore-unknown"
   },
   "dependencies": {
+    "js-yaml": "^4.1.0",
```

Comment:
hi, would you like to manually edit the package.json so the diff is more readable. thanks!